src/connections/DelayedJob.ts

```diff
@@ -0,0 +1,103 @@
+import { useEffect } from "react";
+import { createSelector } from "reselect";
+
+import { bulkUpdateJobs, listDelayedJobs } from "@/generated/v3/jobService/jobServiceComponents";
+import {
+  bulkUpdateJobsFetchFailed,
+  bulkUpdateJobsIsFetching,
+  listDelayedJobsFetchFailed
+} from "@/generated/v3/jobService/jobServicePredicates";
+import { DelayedJobData, DelayedJobDto } from "@/generated/v3/jobService/jobServiceSchemas";
+import { ApiDataStore } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionHook, connectionLoader } from "@/utils/connectionShortcuts";
+
+// --- Delayed Jobs Connection ---
+type DelayedJobsConnection = {
+  delayedJobs?: DelayedJobDto[];
+  isLoading: boolean;
+  hasLoadFailed: boolean;
+};
+
+const delayedJobsSelector = (store: ApiDataStore) => {
+  const delayedJobsMap = store.delayedJobs || {};
+  return Object.values(delayedJobsMap).map(resource => resource.attributes);
+};
+
+const delayedJobsLoadFailedSelector = (store: ApiDataStore) => {
+  return listDelayedJobsFetchFailed(store) != null;
+};
+
+const connectionIsLoaded = ({ delayedJobs, hasLoadFailed, isLoading }: DelayedJobsConnection) => {
+  return (delayedJobs != null && delayedJobs.length > 0) || hasLoadFailed || isLoading;
+};
+
+const delayedJobsConnection: Connection<DelayedJobsConnection> = {
+  load: connection => {
+    const isLoaded = connectionIsLoaded(connection);
+    if (!isLoaded) {
+      listDelayedJobs();
+    }
+  },
+  isLoaded: connectionIsLoaded,
+  selector: createSelector(
+    [delayedJobsSelector, store => delayedJobsLoadFailedSelector(store)],
+    (delayedJobs, hasLoadFailed): DelayedJobsConnection => ({
+      delayedJobs,
+      isLoading: delayedJobs == null && !hasLoadFailed,
+      hasLoadFailed
+    })
+  )
+};
+
+export const useDelayedJobs = () => {
+  const connection = connectionHook(delayedJobsConnection)();
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      listDelayedJobs();
+    }, 1500);
```

Comment:
This will cause this interval to keep refreshing the delayed jobs for as long as the surrounding component is mounted. Is that intentional? It seems like clearing the interval when all the delayed jobs have completed would be good to keep from continuing to hit the jobs endpoint. I'm OK with letting this live to prod if that's complicated to wire up; it's just not ideal and should be cleaned up in a future ticket.

Please either decide to implement an eventual stop like that in this PR, or see that a tech debt ticket is created to clean up in a future release.

Reply:
Yes indeed, it is supposed to be attentive for any job created on any session. 
Nonetheless I think it might not be good. 
So for now the debt ticket will be needed, because we would have to think on the user cases.